pkg/oc/clusterup/coreinstall/kubeapiserver/openshift_apiserver.go

```diff
@@ -36,6 +38,35 @@ func MakeOpenShiftAPIServerConfig(existingMasterConfig string, routingSuffix, ba
 	// hardcode the route suffix to the old default.  If anyone wants to change it, they can modify their config.
 	masterconfig.RoutingConfig.Subdomain = routingSuffix
 
+	// default openshift image policy admission
+	masterconfig.AdmissionConfig.PluginConfig["openshift.io/ImagePolicy"] = &configapi.AdmissionPluginConfig{
```

Comment:
This should be a default (if no plugin config for ImagePolicy is defined), since you're reading existing.

Reply:
@smarterclayton i'm reading but also writing the modified one on disk with this change...

I think since this admission is openshift specific, it should be part of the openshift api server component, right?